### PR TITLE
Matthew getting data

### PR DESCRIPTION
**Pull request by MatthewHiscock**
Changed image names but code is a mishmash of html - not quite sure how to proceed. @mariahhamel, @chriddyp, @NicoleGrondin, @aneda, @mkcor

### CODE REVIEW

File: _posts/2015-03-30-getting-data.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ meta_description: A tutorial on reliable data sources including Data.gov. Plotly
...
 
-You can also [sync Plotly graphs with arbitrary externally sourced data](http://chriddyp.github.io/external-source-plotter/). These graphs can be shared by a URL or embedded in a webpage, and whenever a user views the graph, Plotly fetches the latest data from your web service. Use the [external source plotter](http://chriddyp.github.io/external-source-plotter/) when you don't want to send your user to Plotly's web-app. We have examples documented for SQLite, Flask, Quandl, Google Drive, GitHub Gists, and Dropbox. 
+You can also [sync Plotly graphs with externally sourced data](http://chriddyp.github.io/external-source-plotter/). These graphs can be shared by a URL or embedded in a webpage, and whenever a user views the graph, Plotly fetches the latest data from your web service. Use the [external source plotter](http://chriddyp.github.io/external-source-plotter/) when you don't want to send your user to Plotly's web-app. We have examples documented for SQLite, Flask, Quandl, Google Drive, GitHub Gists, and Dropbox.
```
Comment (mkcor) [inline _posts/2015-03-30-getting-data.md]: "web-app"? I thought we were writing "Web App", unless I missed an update on this (which is totally possible).

Comment (MatthewHiscock) [inline _posts/2015-03-30-getting-data.md]: You're right - I'll change that.

Comment (mkcor) [inline _posts/2015-03-30-getting-data.md]: I would make the first hyperlink go from only "externally sourced data" (in meaning, I totally get it, but in practice and visually, it's better to keep hyperlinked text short).

Comment (MatthewHiscock) [inline _posts/2015-03-30-getting-data.md]: Makes sense.

**Comment by mkcor** [comment]
:+1:

**Pull request merged**